optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/StatisticUtils.java

```diff
@@ -68,8 +69,12 @@ public static String getStandardDeviationString(double[] standardDeviationDouble
         StringBuilder standardDeviationString = new StringBuilder(standardDeviationDoubles.length * 9);
         // Abbreviate to 2 decimals
         // We don't use a local sensitive DecimalFormat, because other Scores don't use it either (see PLANNER-169)
+        DecimalFormatSymbols decimalFormatSymbols = DecimalFormatSymbols.getInstance();
```

Comment:
As long as PLANNER-169 isn't implemented, we want those numbers to be formatted locale-indepent, like java literals (such a "double d = 1.5;"). By default, the Score.toString() also follow the java literal format (similar to Integer.toString() and BigDecimal.toString()).

Java literals use en_US. Notice how we don't write "double d = 1,5;". So I think it needs to be:

    new DecimalFormatSymbols(Locale.US)